test/create_db.sh

```diff
@@ -3,14 +3,37 @@ set -o errexit
 cd $(dirname $0)/..
 source test/db-common.sh
 
-# set db connection for if running in a separate container or not
+
+# set db connection for if running in a seperate container or not
 dbconn="-u root"
 if [[ ! -z "$MYSQL_CONTAINER" ]]; then
 	dbconn="-u root -h 127.0.0.1 --port 3306"
 fi
 
-# Drop all users to get a fresh start
-mysql $dbconn < test/drop_users.sql
+#s/\.[0-9][0-9]*$//
+
+DB_VERSION=$(  mysql $dbconn -e "status;" | grep "Server version" | sed -e 's/-Maria.*//
```

Comment:
I meant to say that we would avoid parsing the `status;` output entirely and just look for 10.1 vs 10.0.